test/parallel/test-v8-serdes.js

```diff
@@ -20,6 +20,8 @@ const objects = [
   circular
 ];
 
+const hostObject = new (process.binding('js_stream').JSStream)();
```

Comment:
🤔 I thought this exploded last time I tried it.

Reply:
You mean, constructing a `JSStream` like that? It seems to work fine to me